roles/rsyslog/templates/rsyslog.conf.j2

```diff
@@ -0,0 +1,6 @@
+# {{ ansible_managed }}
+# rsyslog version is {{ __rsyslog_version }}.
```

Comment:
I don't think we want to have `__rsyslog_version` here - having it here will mean the file will be updated unnecessarily every time rsyslog is updated to a new version

Reply:
It was controversial and never liked by @pcahyna. :)
With the change in this PR, CI tests stop referring it. So, if it is ok to check the version using the internal __rsyslog_version in the CI test playbook, I'm happy to delete it.

Reply:
Can you show me how you are using `__rsyslog_version` ?

Reply:
In the test playbook, __rsyslog_version is set to "N/A" first.
https://github.com/nhosoi/logging/blob/remove_rsyslog_default/tests/tests_enabled.yml#L4-L5
Then,after Logging role was run, it checks the value is no longer "N/A" or "0.0.0" that indicates something is wrong happened on the rsyslog package.
https://github.com/nhosoi/logging/blob/remove_rsyslog_default/tests/tests_enabled.yml#L14-L16

Reply:
ok - then yes, please remove from templates/rsyslog.conf.j2